Extensions/PathfindingBehavior/tests/legacypathfindingruntimebehavior.spec.js

```diff
@@ -1,6 +1,8 @@
 // @ts-check
 describe('gdjs.PathfindingRuntimeBehavior', function () {
   // limit tests cases on the legacy collision methods.
+  // Not that the legacy collision mode is still the only mode that exists
+  // because the new one were never merged.
```

Comment:
not related to this PR, but why has it never been merged?

Reply:
The obstacle detection is a big issue of the behavior because it only handles default hiboxes without rotation.
The new method was doing true collision checks instead of boxes check. There was a risk that it would be too slow and we would need yet another collision method. So we decided to wait for a better solution.
The better solution could be a NavMesh, but it has the issue to take a long time to build (maybe a worker could solve this) and I haven't implemented incremental updates so it's not really usable.

Reply:
ok thanks 👍